tensorflow/core/grappler/optimizers/meta_optimizer.cc

```diff
@@ -165,10 +165,26 @@ bool MemoryOptimizerEnabled(
   return mem_opt_type != RewriterConfig::NO_MEM_OPT;
 }
 
+Status GetGraphDevice(const GraphDef& g_def, std::set<std::string>& devices) {
```

Comment:
Output parameters must be taken by pointer.

Reply:
Done